<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LanguageManagerClass</class>
 <widget class="QMainWindow" name="LanguageManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LanguageManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidgetProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>421</width>
      <height>471</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Project</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lng-File</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>EXCEL files</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadExcel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load EXCEL file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDeleteExcel">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete from list</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetExcelFiles">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>741</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>FileName</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Path</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>721</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Output LNG file and path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditLNGFile">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLNGFile">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>480</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCompile">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Compile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBuildAll">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>550</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Build all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddNewProject">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add new project</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarCompile">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>530</y>
      <width>471</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_solution"/>
    <addaction name="actionLoad_solution"/>
    <addaction name="actionSave_solution"/>
    <addaction name="actionOverwrite_solution"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Project"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionLanguage"/>
    <addaction name="actionDefault"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoad_solution">
   <property name="text">
    <string>Load solution</string>
   </property>
  </action>
  <action name="actionLanguage">
   <property name="text">
    <string>Language</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionSave_solution">
   <property name="text">
    <string>Save solution</string>
   </property>
  </action>
  <action name="actionNew_solution">
   <property name="text">
    <string>New solution</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionOverwrite_solution">
   <property name="text">
    <string>Overwrite solution</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
